src/windows/top_toolbar.c

```diff
@@ -1461,10 +1462,11 @@ static void window_top_toolbar_scenery_tool_down(short x, short y, rct_window *w
 	case SCENERY_TYPE_SMALL:
 	{
 		int quantity = 1;
-		if (window_scenery_is_build_cluster_tool_on) {
-			quantity = 35;
+		bool isCluster = window_scenery_is_build_cluster_tool_on && network_can_perform_action(network_get_current_player_group_index(), GAME_COMMAND_TOGGLE_SCENERY_CLUSTER);
+		if (isCluster) {
+			quantity = 36;
```

Comment:
all you have to do is get rid of the game command and make STR_ACTION_TOGGLE_SCENERY_CLUSTER = {-2}, and use -2 in network_can_perform_action, look at STR_ACTION_CHAT